source/dom/resolution.js

```diff
@@ -4,13 +4,11 @@
 		_riRatio,
 		_screenType,
 		_screenTypes = [ {name: 'standard', pxPerRem: 16, width: scope.innerWidth,  height: scope.innerHeight, aspectRatioName: 'standard'} ],	// Assign one sane value in case defineScreenTypes is never run.
-		_screenTypeObject;
+		_screenTypeObject,
+		_oldScreenType;
 
 	var getScreenTypeObject = function (type) {
 		type = type || _screenType;
-		if (type == _screenType && _screenTypeObject) {
-			return _screenTypeObject;
-		}
```

Comment:
Why was this optimization removed? (Reasoning) It prevents an array search if the request is the same as the current screenType.